packages/ui-components/src/Input/CommonInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import classNames from "classnames/bind";
+import "./input.module.scss";
+
+export interface CommonInputProps {
+  id?: string;
+  // this should be the element containing the input
+  fieldInput?: JSX.Element;
+  className?: string;
+  disabled?: boolean;
+  help?: string | JSX.Element;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  error?: any;
+  inline?: boolean;
```

Comment:
I think it could, but it's most commonly used in SingleLineTextInputs, I haven't seen it used anywhere else. I couldn't find documentation about which Inputs it applies to so I assumed all -- perhaps not a correct assumption?